tests/main.rs

```diff
@@ -250,3 +250,63 @@ async fn test_max_by_and_min_by() {
     - +---------------------+
     "###);
 }
+
+#[tokio::test]
+async fn test_kurtosis_pop() {
+    let mut execution = TestExecution::new().await.unwrap().with_setup(TEST_TABLE).await;
```

Comment:
There were additional tests too in https://github.com/apache/datafusion/pull/12647

https://github.com/apache/datafusion/pull/12647/files#diff-a60d69cf923e18b0532e7ba3470c4b728e6b00230b3ea242f3fbe95e899c5ffeL5866

Maybe it is time to consider adding sqllogictest to this repo too 🤔  (not as part of this PR)

Reply:
+1 I was thinking the same. Plus it kind of hazzle to assert those table output snapshots.

Reply:
I would love to work on adding sqllogictest to this project

@dharanad  @alamb